pkg/server/log.go

```diff
@@ -34,18 +35,22 @@ func (f *logFormatter) Format(entry *log.Entry) ([]byte, error) {
 }
 
 func (s *Server) initLogs() error {
-	filePath := s.config.LogFilePath
-
-	file, err := os.OpenFile(filePath, os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		return err
+	filePath := s.config.LogFile.Path
```

Comment:
![Screenshot 2024-11-15 at 11 59 41](https://github.com/user-attachments/assets/ed835271-bd30-4ab5-a1be-fa2f4890584a)

Lets add a nil check here. Currently, because of the breaking change, the user won't even be able to start the server.
If we add the nil check, users would be able to address the breaking change by starting the server, running `server configure` and editing the props through the UI.